versions.gradle

```diff
@@ -24,6 +24,7 @@ ext.deps = [:]
 def versions = [:]
 // GeckoView versions can be found here:
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
+versions.gecko_view_release = "115.0.20230710165010"
```

Comment:
I've mentioned in https://github.com/Igalia/wolvic/pull/820#issuecomment-1634702715 that now we shouldn't have a gecko version higher than 105 as almost everything is broken (bookmarks, history, and extensions ...).

```suggestion
versions.gecko_view_release = "105.0.20221007134813"
```

![Screenshot from 2023-07-31 20-15-30](https://github.com/Igalia/wolvic/assets/43995067/098c167f-a9af-49a1-9552-ea3b28214244)

Reply:
Oh sure, it's just that I was experimenting with 115 ESR. In any case it isn't the point of the patch so I'm obviously fine with using 105 instead